client/my-sites/site-settings/composing/after-the-deadline.jsx

```diff
@@ -216,22 +215,12 @@ class AfterTheDeadline extends Component {
 				<QueryJetpackConnection siteId={ selectedSiteId } />
 
 				<div className="composing__module-settings site-settings__child-settings">
-					<div className="composing__info-link-container site-settings__info-link-container">
-						<InfoPopover position="left">
-							{ translate(
-								'Checks your content for correct grammar and spelling, ' +
-									'misused words, and style while you write.'
-							) }{' '}
-							<ExternalLink
-								href="https://jetpack.com/support/spelling-and-grammar/"
-								icon={ false }
-								target="_blank"
-							>
-								{ translate( 'Learn more' ) }
-							</ExternalLink>
-						</InfoPopover>
-					</div>
-
+					<SupportInfo
+						text={ translate(
+							'Checks your content for correct grammar and spelling, misused words, and style.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Checks your content for correct grammar and spelling, misused words, and style while you write.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Yup, I'm with the bot on this one 😄

Reply:
Done, ty :-)